04.class/destroy_enquirer.js

```diff
@@ -0,0 +1,23 @@
+const DbOperator = require("./db_operator.js");
+const dbOperator = new DbOperator();
+const sqlite3 = require("sqlite3").verbose();
+const db = new sqlite3.Database("memo_app.sqlite3");
+const { Select } = require("enquirer");
+module.exports = class DestroyEnquirer {
```

Comment:
`require`している部分はは一つの塊にした方が見やすいと思いますし、処理の種類によっていい感じに空行を入れてあげるとさらに見やすいと思います。特に、`require`が終わった後とか、クラス定義の前とかは空行必須です。こんな感じです。

```suggestion
const DbOperator = require("./db_operator.js");
const sqlite3 = require("sqlite3").verbose();
const { Select } = require("enquirer");

const dbOperator = new DbOperator();
const db = new sqlite3.Database("memo_app.sqlite3");

module.exports = class DestroyEnquirer {
```